fw-child/resources/js/map.js

```diff
@@ -1350,39 +1351,52 @@
         options = plugin.options,
         item = plugin.item;
 
-      // todo: implement behaviour for building_climate_zones
-      let layer_name = $(document).cdc_app(
-        'maps.get_layer_name',
-        options.query,
+      let default_scheme_element = item.find(
+        '#display-scheme-select .dropdown-item[data-scheme-id="default"]',
       );
-      $.getJSON(
-        geoserver_url +
+
+      if (special_variables.hasOwnProperty(options.var_data.slug)) {
+        default_scheme_element.data('scheme-colours', special_variables[options.var_data.slug].colormap.colours);
+        default_scheme_element.data('scheme-quantities', special_variables[options.var_data.slug].colormap.quantities);
+        plugin.update_scheme();
+
+        if (typeof callback == 'function') {
```

Comment:
Use 3 "=" to be consistent with other similar "if"
```suggestion
        if (typeof callback === 'function') {
```